Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -772,6 +821,14 @@ public void Deconstruct()
             furniture.Jobs.CancelAll();
         }
 
+        if (deconstructInventory != null)
+        {
+            foreach (Inventory inv in deconstructInventory)
+            {
+                World.Current.inventoryManager.PlaceInventory(Tile, inv);
```

Comment:
My solution for the landing pad was to (create and) use the 

```
ile tile = WorldController.Instance.World.GetFirstTileWithNoInventoryAround(6, (int)tradingCoordinates.x, (int)tradingCoordinates.y, (int)tradingCoordinates.z);
Inventory inv = new Inventory(tradeItem.Type, tradeItem.TradeAmount, tradeItem.TradeAmount);
WorldController.Instance.World.inventoryManager.PlaceInventory(tile, inv);
```

Not the best yet, but at least it give more options to the code